jgrapht-core/src/main/java/org/jgrapht/alg/scoring/AlphaCentrality.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * (C) Copyright 2016-2018, by Dimitrios Michail and Contributors.
```

Comment:
Copyright should be you as original author, with year 2018-2018 since the file is new.